app/models/manageiq/providers/google/inventory/collector/network_manager.rb

```diff
@@ -150,9 +150,6 @@ def get_vm_id_cached(zone, instance)
     begin
       @vm_cache.store_path(zone, instance, connection.get_server(instance, zone).id)
     rescue Fog::Errors::Error, ::Google::Apis::ClientError => err
-      m = "Error during data collection for [#{manager&.name}] id: [#{manager&.id}] when querying link for vm_id: #{err}"
```

Comment:
Let's add a comment here explaining why this is a no-op.

Reply:
(Oops, commented on the wrong side. Anyway, I think you know what I meant.)

Reply:
:+1: done